### PR TITLE
Change focus upon 'enter' input

### PR DESCRIPTION
**Pull request by Alexei-Savich**
1) Change the focus of the current input field when 'enter' is detected:
- Login screen
- Registration screen
- Add spending screen
2) Add email validation upon registration

### CODE REVIEW

File: app/src/main/java/com/rebalance/ui/component/main/ExpandableList.kt [hunk 1]
```diff
@@ -119,14 +125,50 @@ fun ExpandableList(
...
+                                                        imgBase64.size
+                                                    ), 100, 100, false
+                                                ).asImageBitmap(),
+                                                contentDescription = "Expanse image as an icon"
+                                            )
+                                        } else {
+                                            Icon(EvaIcons.Fill.Image, "Image placeholder")
+                                        }
+
+                                    }
```
Comment (DemyDeni) [inline app/src/main/java/com/rebalance/ui/component/main/ExpandableList.kt]: Move repeating code to separate component

File: app/src/main/java/com/rebalance/ui/component/main/ExpandableList.kt [hunk 2]
```diff
@@ -119,14 +125,54 @@ fun ExpandableList(
...
+                                    } else {
+                                        IconButton(onClick = {
+                                            alertUser(
+                                                "No image for this expense",
+                                                context
+                                            )
+                                        }) {
+                                            Icon(EvaIcons.Fill.Image, "Image placeholder")
+                                        }
+                                    }
```
Comment (demydeni) [inline app/src/main/java/com/rebalance/ui/component/main/ExpandableList.kt]: Move repeating code to separate component

File: app/src/main/java/com/rebalance/utils/UtilsUI.kt [hunk 3]
```diff
@@ -13,3 +31,59 @@ fun alertUser(message: String, context: Context) {
         ).show()
     }
 }
+
+@Composable
+fun displayExpenseImage(
```
Comment (demydeni) [inline app/src/main/java/com/rebalance/utils/UtilsUI.kt]: Please move it to ui/components/main package as separate component.

Comment (VladKhrlmv) [reply app/src/main/java/com/rebalance/utils/UtilsUI.kt]: done

**Review by demydeni — APPROVED** [review]

**Pull request merged**